resources/tiers.md

```diff
@@ -0,0 +1,89 @@
+
+The objective of Meilisearch's Integrations Team is to **ease Meilisearch usage to as many developers as possible**.
+
+Given this statement, **tiers** are nothing more than a __group of SDKs/integrations__ where the Integrations Team can fairly divide their attention. Unfortunately is not possible to keep all the SDKs 100% updated against the new Meilisearch features from Meilisearch releases all the time. The Integrations Team is a polyglot small team that maintains 30+ different integrations and tools, so it is humanly impossible to consistently meet the Meilisearch quality standards.
+
+The ultimate goal of introducing tiers is to allow the team to have more time to invest in new projects/integrations to impact even more users.
+
+To measure the impact of an SDK, the data come from different places like [telemetry](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html) and GitHub's stars, forks, and watchers.
+
+It is worth remembering that this priority is just a concept introduced by the team that aims to give a **decision weight** before investing time in some issues of the SDKs. So it does not mean they will never work on feature requests from the bottom tiers anymore, but it says they will prioritize the top tiers most of the time transparently.
+
+:warning: The team will still keep all officially maintained SDKs **working**. :warning:
+
+## Tier #1
+
+The SDKs in group #1 have absolute priority over the team's time. For example, issues are visited before issues are present in tier #2.
+It is worth remembering that this does not mean open issues are addressed and fixed promptly. Every issue needs time to be understood/confirmed/worked on and consequently closed.
+
+All new Meilisearch features that make sense for those SDKs will be available on the release day.
+
+## Tier #2
+
+Tier #2 SDKs are the emergent ones. The Integrations Team believes they can move further and are betting on them.
+In any case, since they bring less value than tier #1, the team will only spend the time required to keep these SDKs working and do the necessary maintenance code and code cleanup.
+
+Only in urgent cases where, for example, the SDK is no longer working issues from the SDKs in group #2 prioritized over Tier #1.
+
+## Tier #3
+
+The SDKs in group #3 have relatively low user adoption compared to groups #2 and #1. They are probably new or have a small community behind the tech stack in the first place.
+As with tier #2, SDKs in this group are prioritized only in critical cases, and they are not going to receive active improvements or maintance from the integrations team.
+
+This is a very special group because the Integrations Team really need help from the community here.
+
+
+## Tiers list
+
+Integration | Tier |
+-------------|------|
+Javascript | #1 |
+PHP | #1 |
+Instant Meilisearch | #1 |
+Python | #1 |
+Go | #1 |
+Strapi | #2 |
+Ruby | #2 |
+Rails | #2 |
+.NET | #2 |
+Rust | #2 |
+Symfony | #3 |
+docs-searchbar.js | #3 |
```

Comment:
Is there a reason to have `docs-searchbar.js` being tier 3? 

That's potentially something we will want to maintain / replace actively at some point because It's an excellent opener to have a tool that works to get many projects moving from Algolia to Meilisearch for their documentation.
 
What's the rationale for prioritizing a severe issue on tier 3?

Reply:
Not really. It is just because we don't have the User Agent there!